pkg/resourceinterpreter/configurableinterpreter/configmanager/manager.go

```diff
@@ -32,7 +32,7 @@ type ConfigManager interface {
 
 // interpreterConfigManager collects the resource interpreter customization.
 type interpreterConfigManager struct {
-	initialSynced *atomic.Value
+	initialSynced *atomic.Bool
```

Comment:
I think here don't need to use the pointer. Since the zero value isn't `nil` but `false`.
What do you think?

Reply:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/admission/configuration/mutating_webhook_manager.go#L51




`k/k` is also the pointer used.

Reply:
> k/k is also the pointer used.

I know, but obviously, we didn't think about it when borrowing the idea from k/k.  And we can contribute back to k I guess.

Reply:
get it.